### PR TITLE
Add Community Guides footer with Render

### PR DESCRIPTION
**Pull request by anurag**
Adding a Community Guides footer that can be used for other platforms. Added Render to it; Nanobox isn't accepting new signups so I didn't include it.

### CODE REVIEW

File: guides/deployment/deployment.md [hunk 1]
```diff
@@ -116,4 +116,7 @@ And that's it. Next you can use one of our official guides to deploy:
 
   * [with Elixir's releases](releases.html)
   * [to Gigalixir](gigalixir.html), an Elixir-centric Platform as a Service (PaaS)
-  * and [to Heroku](heroku.html), one of the most popular PaaS
+  * and [to Heroku](heroku.html), one of the most popular PaaS.
+
+## Community Deployment Guides
```
Comment (josevalim) [inline guides/deployment/deployment.md:121]:
```suggestion
## Community Deployment Guides

```

File: guides/deployment/deployment.md [hunk 2]
```diff
@@ -116,4 +116,8 @@ And that's it. Next you can use one of our official guides to deploy:
...
+
+* [Render](https://render.com) has first class support for Phoenix applications, with no connection limits, no forced restarts and full support for Elixir clusters. Check out their guides for hosting Phoenix with [Mix releases](https://render.com/docs/deploy-phoenix), [Distillery](https://render.com/docs/deploy-phoenix-distillery), and as a [Distributed Elixir Cluster](https://render.com/docs/deploy-elixir-cluster).
```
Comment (josevalim) [inline guides/deployment/deployment.md:123]:
```suggestion
  * [Render](https://render.com) has first class support for Phoenix applications. There are guides for hosting Phoenix with [Mix releases](https://render.com/docs/deploy-phoenix), [Distillery](https://render.com/docs/deploy-phoenix-distillery), and as a [Distributed Elixir Cluster](https://render.com/docs/deploy-elixir-cluster).
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by anurag** [comment]
> 💚 💙 💜 💛 ❤️

Thanks!

**Pull request merged**